hazelcast/src/main/java/com/hazelcast/map/impl/operation/EvictAllBackupOperation.java

```diff
@@ -37,8 +37,6 @@ public EvictAllBackupOperation(String name) {
 
     @Override
     public void run() throws Exception {
-        clearLocalNearCache();
```

Comment:
I couldn't find a test for this scenario, so I wrote one: https://github.com/hazelcast/hazelcast/pull/8869/files#diff-1255ed0d931aed11a9aa83ddb3864b8fR799

:+1: